ui/admin/tests/acceptance/sessions/list-test.js

```diff
@@ -229,6 +229,21 @@ module('Acceptance | sessions | list', function (hooks) {
     assert.dom('tbody tr').exists({ count: 1 });
   });
 
+  test('users filter is hidden if no users', async function (assert) {
```

Comment:
small suggestion: perhaps these two test case titles can be updated to be similar to the one in the targets folder below